tests/func/experiments/test_show.py

```diff
@@ -269,6 +270,10 @@ def test_show_filter(
     included,
     excluded,
 ):
+    from contextlib import contextmanager
```

Comment:
For now, I think this workaround of setting console width is okay. But I'd like to see making columns collapse configurable and also get rid of this test with simpler versions, this is testing too much. :)

Reply:
Yeah, but we had mixed too much in this PR, I‘d prefer opening a new `columns collapse configuration` ticket

Reply:
I am not asking you to work on it, I was just making a note: :)